Amplify/Core/Configuration/AmplifyConfiguration.swift

```diff
@@ -25,6 +26,9 @@ public struct AmplifyConfiguration: Codable {
     /// Configurations for the Amplify API category
     let api: APICategoryConfiguration?
 
+    /// Configurations for the Amplify Auth category
+    let auth: AuthCategoryConfiguration?
+
```

Comment:
On line 32, the comment needs to be updated so that it says Amplify Datastore Category 😛